docs/rules.md

```diff
@@ -890,6 +877,13 @@ A POST operation's operationId should contain the verb indicated at the end of t
 
 Please refer to [post-operation-id-contains-url-verb.md](./post-operation-id-contains-url-verb.md) for details.
 
+### PostResponseCodes
+
+Synchronous POST operations must have one of the following combinations of responses - 200 and default ; 204 and default. They also must not have other response codes.
+Long-running POST operations must have responses with 202 and default return codes. They must also have a 200 return code if only if the final response is intended to have a schema, if not the 200 return code must not be specified. They also must not have other response codes.
+
```

Comment:
Minor - should we show an example for valid and invalid responses?

Reply:
Rules.md has autogenerated code, these changes arent mine I did the merge from main these changes might be because merge.